docs/cli/README.md

```diff
@@ -7,6 +7,7 @@ To do that, pass the `--testnet` flag with the actual command:
 ```shell
 swap --testnet <SUBCOMMAND>
 ```
+
```

Comment:
this commit doesn't seems to only make new line. I think you can drop this and reword the commit to use this commit message. I am happy to take over if the back and forth is getting tedious.

Reply:
Apologies,  I'm such a newbie and got lost a bit in git - my edits are done through Github and not from my local machine. My intent was to add a line of  comment
>Take note that applying `--testnet` results in transacting on Bitcoin testnet and on Monero stagenet, not on Monero testnet.

If you merge it now wouldn't this comment be displayed below 
```shell
swap --testnet <SUBCOMMAND>
``` as I intended? I run `dprint check` and it passed it favourably. @rishflab you're welcome to take over, apologies again for the nuisance.

Reply:
no nuisance at all : ) everyone was a newbie at some point. I will squash the commits no probs